### PR TITLE
Delete all acme challenge records

### PR DESCRIPTION
**Pull request by c-e-brumm**
After completing an acme challenge all acme challenge records for the same hosted zone are deleted. This will ensure that no records are left behind in a zone. And if records are left behind from before they will be cleaned up when the zone is used next.

### CODE REVIEW

File: pkg/awsclient/client.go [hunk 1]
```diff
@@ -78,6 +79,52 @@ func (c *awsClient) ListResourceRecordSets(input *route53.ListResourceRecordSets
...
+		HostedZoneId: &hostedZone,
+	}
+	return input
+}
+
+// CreateR53TXTRecordChange creates an route53 Change object for a TXT record with a given name
+// and a given action to take.
+func CreateR53TXTRecordChange(name *string, action string, value *string) route53.Change {
+	change := route53.Change{
+		Action: aws.String(route53.ChangeActionDelete),
```
Comment (dak1n1) [inline pkg/awsclient/client.go]: I see `action` passed in, but not used. Should it go on this line instead of `ChangeActionDelete`?

Comment (c-e-brumm) [reply pkg/awsclient/client.go]: Good catch. It looks like I forgot to write the switch/case statement for setting dynamic actions.

Comment (c-e-brumm) [reply pkg/awsclient/client.go]: Updated, now checks the action passed and sets the correct r53 action.

**Pull request merged**